snapcraft.yaml

```diff
@@ -70,7 +70,7 @@ parts:
 
     source: https://git.launchpad.net/curtin
     source-type: git
-    source-commit: 237053d9d18916dd72cf861280474d4df0e9fd24
+    source-commit: 79d831871d32e396326106b5f05ba144e61badaa
```

Comment:
Update when the relevant curtin MP gets merged.

Reply:
Is what you need merged at this point?

Reply:
I just hit merge for the curtin part. I originally wanted to wait for the FFe decision but we can still exclude the changes from the stable branch in case it gets rejected.